aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,197 @@
+  
+var app = builder.Build();
+
+app.Run();
+```
+
+## Configuring authorization policies in minimal applications
+
+While authentication is used to identify and validate the identity of users against an API, authorization is used to validate and verify access to resources in an API and is faciliated by the `IAuthorizationService` registered by the `AddAuthorization` extension method. In the scenario below, we want to add an `/hello` resource that requires a user to present an `is_admin` claim with a `greetings_api` scope.
+
+Configuring authorization requirements on a resource is a two-step process that requires:
+
+1. Configuring the authorization requirements in a policy globally.
+2. Applying individual policies to resources.
+
+In the code sample below, the  `AddAuthorizationBuilder` is invoked which:
+
+- Adds authorization-related services to the DI container.
+- Returns an `AuthorizationBuilder` that can be used to directly register authentication policies.
+
+The code creates a new authorization policy, named `policy_greetings`, that encapsulate two authorization requirements:
+
+- A role-based requirement that the user fall under the `admin` role.
+- A claim-based requirement that the user provide a `greetings_api` scope.
+
+Finally, the `admin_greetings` policy is provided as a required policy to the `/hello` endpoint.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthorizationBuilder()
+  .AddPolicy("admin_greetings", policy => 
+		policy
+			.RequireRole("admin")
+			.RequireScope("greetings_api"))
+
+var app = builder.Build();
+
+app.MapGet("/hello", () => "Hello world!")
+  .RequireAuthorization("admin_greetings");
+
+app.Run();
+```
+
+## Using `dotnet user-jwts` to improve development time testing
+
+Throughout this article, we've used an application configured with JWT-bearer based authentication as a sample. JWT bearer-based authentication requires that clients present a token in the request header to validate their identity and claims. Typically, these tokens are issued by a central authority, like an identity server.
+
+However, when developing locally, it is helpful to generate these tokens locally and for testing. To that end, the `dotnet user-jwts` tool is helpful for creating bearer tokens.
+
+```dotnetcli
+dotnet user-jwts create
+```
+
+> [!NOTE]
+> When invoked on a project, the tool will automatically add the authentication options matching the generated token to appsettings.json.
+
+Tokens can be configured with a variety of customizations. For example, to create a token provides the `admin` role and `greetings_api` scope expected by the authorization policy above:
+
+```dotnetcli
+dotnet user-jwts create --scope "greetings_api" --role "admin"
+```
+
+The generated token can then be sent as part of the header in the testing tool of choice. For example, with curl:
+
+```dotnetcli
+curl -i -H "Authorization: Bearer {token}" https://localhost:{port}/hello
```

Comment:
nit: `"Authorization: Bearer <token>"`
I think `<something>` is the more common way to express "replace me"

Reply:
`{}` feels like a better options since `<>` might be overloaded with the tag syntax that is used elsewhere in the docs and `{}` follows more nicely with the template string syntax.

@Rick-Anderson What's the standard style here?

Reply:
> `{}` feels like a better options since `<>` might be overloaded with the tag syntax that is used elsewhere in the docs and `{}` follows more nicely with the template string syntax.
> 
> @Rick-Anderson What's the standard style here?

I agree `{}` is better and more obvious. We've had a couple of complaints about `<>`. cc @tdykstra

Reply:
The [contributor guide calls for angle brackets](https://review.learn.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=main#placeholders), adding that curly braces are OK only for languages like HTML where angle brackets have a special meaning.

>We discourage curly braces { } for use as syntactical placeholders. Readers can confuse curly brace placeholders with the same notation used in:
>
>Replaceable text
> * Format strings
> * String interpolation
> * Text templates
> * Similar programming constructs
>
>Consider using curly braces instead of angle brackets when angle brackets are normally used in the syntax of a language, such as in XML or HTML.

Reply:
> The [contributor guide calls for angle brackets](https://review.learn.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=main#placeholders), adding that curly braces are OK only for languages like HTML where angle brackets have a special meaning.
> 

The `contributor guide`  needs to be updated to include C#

Reply:
> The contributor guide needs to be updated to include C#

Would the C# guide be updated to recommend `{}`. IMO, that makes more sense given the proximity to template strings in C#.

Reply:
C# is a language where `<>` is part of the language.